test/init.js

```diff
@@ -124,8 +124,16 @@ const buildPlayer = async (dnasToInstall, graphQLAPIOptions) => {
           manifest_version: '1',
           roles: dnasToInstall.map((name) => ({
             name: `hrea_${name}_1`,
+            // https://docs.rs/holochain_types/0.1.0-beta-rc.1/src/holochain_types/app/app_manifest/app_manifest_v1.rs.html#165-180
```

Comment:
adding this fixed the uniqueness of the dna, because of complying with this code:
https://github.com/holochain/holochain/blob/a28eff8dbd688f59e2d16347185040336a0177f0/crates/holochain_types/src/app/app_manifest/app_manifest_v1.rs#L196-L202

tricksy.